### PR TITLE
Mareo

### PR DESCRIPTION
**Pull request by mareo1997**

### CODE REVIEW

File: src/main/java/com/revature/model/Pokemon.java [hunk 1]
```diff
@@ -1,18 +1,156 @@
...
 	private String type1;
 	
+	@Column(name="type1")
 	private String type2;
 	
+	@Column(name="ability", nullable = false)
 	private String ability;
 	
+	@Column(name="category", nullable = false)
+	private String category;
```
Comment (quincy-roman) [inline src/main/java/com/revature/model/Pokemon.java:29]: Not quite sure what this is

File: src/main/java/com/revature/model/Role.java [hunk 2]
```diff
@@ -1,9 +1,87 @@
...
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "Role")
 public class Role {
-	
+
```
Comment (quincy-roman) [inline src/main/java/com/revature/model/Role.java:13]: Trainer doesn't need a role. It won't break anything, but it's an unnecessary foreign key. Let me know if you want to keep it for any reason.

File: src/main/java/com/revature/model/Pokemon.java (hunk 1 above)
Comment (mareo1997) [reply src/main/java/com/revature/model/Pokemon.java:29]: Category is the type of pokemon like how sceptile is the forest pokemon

File: src/main/java/com/revature/model/Role.java (hunk 2 above)
Comment (mareo1997) [reply src/main/java/com/revature/model/Role.java:13]: Ill take it out

**Review by quincy-roman — APPROVED** [review]

**Pull request merged**